guides-and-tutorials/running-a-signer/how-to-monitor-signer.md

```diff
@@ -0,0 +1,165 @@
+# How to Monitor a Signer
+
+We will use [Grafana Cloud](https://grafana.com) to observe and monitor both the
+Signer and its corresponding Stacks node.
+
+## Requirements
+
+Grafana's application observability docs have a [great
+quick-start](https://grafana.com/docs/grafana-cloud/monitor-applications/application-observability/). We will use:
+
+- Grafana Cloud to collect metrics and visualize them.
+- Grafana Alloy, on the Signer host, to push the metrics.
+
+### Creating a Grafana Cloud account
+
+Before we begin, create a [Grafana
+Cloud](https://grafana.com/docs/grafana-cloud/monitor-applications/application-observability/grafana-cloud/) account (they offer a free tier that you can use).
+
+Once done, access your dashboard, click on "Connections", "Add new connection",
```

Comment:
i think the commas here don't work - maybe something like this would read a little better?
```
... access your dashboard and click in "Connections" -> "Add new connection" ...
```

or some other technique to break the line up:
```
...click on :
1. Connections
2. Add new connection
...
```

The text is fine, it's the punctuation here that feels a little off

Reply:
Thanks, addressed in ac4d1cfa9